src/Uno.UI.Tests/Windows_UI_Xaml/Given_StaticResource.cs

```diff
@@ -147,6 +147,8 @@ public void When_Inherited_In_Template_Applied()
 			control.InlineTemplateControl.ApplyTemplate();
 			control.TemplateFromResourceControl.ApplyTemplate();
 
+			control.ForceLoaded();
```

Comment:
Is this necessary? Shouldn't ApplyTemplate() be sufficient?

Reply:
It was required in this case. I didn't investigate why exactly.